include/util/web_mercator.hpp

```diff
@@ -140,13 +140,19 @@ inline FloatCoordinate toWGS84(const FloatCoordinate &mercator_coordinate)
 }
 
 // Converts a WMS tile coordinate (z,x,y) into a wgs bounding box
-inline void xyzToWGS84(
-    const int x, const int y, const int z, double &minx, double &miny, double &maxx, double &maxy)
-{
-    minx = x * TILE_SIZE;
-    miny = (y + 1.0) * TILE_SIZE;
-    maxx = (x + 1.0) * TILE_SIZE;
-    maxy = y * TILE_SIZE;
+inline void xyzToWGS84(const int x,
```

Comment:
wasn't this formatted or are you using a different version than the rest?
At least this looks like it is only caused by the additional mercator_buffer. Might be all correct, though.